crates/release_plz/src/log.rs

```diff
@@ -1,20 +1,30 @@
 use tracing_log::LogTracer;
-use tracing_subscriber::{EnvFilter, FmtSubscriber};
+use tracing_subscriber::{filter, layer::SubscriberExt, EnvFilter, FmtSubscriber};
 
 /// Use `info` level by default, but you can customize it with `RUST_LOG` environment variable.
 pub fn init(verbose: bool) {
     let subscriber = {
+        let env_verbose = std::env::var("RUST_LOG").is_ok() || verbose;
```

Comment:
I don't like that verbose depends on the RUST_LOG env variable 🤔

I would simply use `verbose` instead of `env_verbose`.